mcs/class/corlib/System.Security.Cryptography.X509Certificates/X509Helper.cs

```diff
@@ -50,6 +50,23 @@ internal static void InstallNativeHelper (INativeCertificateHelper helper)
 		}
 
 #if !MOBILE
+		[SecurityPermission (SecurityAction.Demand, UnmanagedCode = true)]
+#endif
+		public static X509CertificateImpl InitFromHandle (IntPtr handle)
+		{
+#if MONO_FEATURE_APPLETLS
+			var variable = Environment.GetEnvironmentVariable ("MONO_TLS_PROVIDER");
+			bool useAppleTls = string.IsNullOrEmpty (variable) || variable == "default" || variable == "apple"; // On Platform.IsMacOS default is AppleTlsProvider
+			if (System.Environment.IsMacOS && useAppleTls)
+				return InitFromHandleApple (handle);
+#endif
+#if !MOBILE
```

Comment:
You should use !ONLY_APPLETLS for the rest which I guess you introduced for that